app/routes.py

```diff
@@ -1,2 +1,248 @@
+    if video is None:
+        return make_response("Video does not exist",404)    
+    else:
+        db.session.delete(video)
+        db.session.commit()
+
+        return {
+                "id":video.id
+        },200
+
+
+@rental_bp.route("check-out",methods=["POST"])
+def check_out():
+    request_body=request.get_json()
+    try:
+        video_id= int(request_body["video_id"]) #make this video_id string into an integer 
+
+        customer_id= int(request_body["customer_id"]) #make this customer_id string into an integer
+    except ValueError or KeyError:
+        return make_response({"error":"Bad Request"},400)
+
+    
+    #Checks to see if the number of videos is more than 0 in inventory so we can rent it out
+    video=Video.query.get(request_body["video_id"]).available_inventory()
+    
+    if video==0:
+        return make_response({"error":"Bad Request"},400)
+    
+    #if all conditions are met, we return the rental status
+    else:
+        new_rental = Rental(customer_id=customer_id, video_id=video_id)
+        
+        db.session.add(new_rental)
+        db.session.commit()
+        
+        return new_rental.rental_check_out()
+
+
+
+@rental_bp.route("check-in",methods=["POST"])
+def check_in():
+   
+    request_body=request.get_json()
+    
+    try:
+        video_id= int(request_body["video_id"]) #make this video_id string into an integer 
+
+        customer_id= int(request_body["customer_id"]) #make this customer_id string into an integer
+    except ValueError or KeyError:
+        return make_response({"error":"Bad Request"},400)
+
+    #if rental doesnt exist, then return error
+    rental = Rental.query.get((customer_id,video_id))
+    if rental is None:
+        return make_response({"error":"Bad Request"},400)
+   
+    db.session.delete(rental)
+    rental_return=rental.rental_check_in()
+    db.session.commit()
+
+    
+    print("rental_return",rental_return)
+    return rental_return
+        
+    
+
+
+@video_bp.route("/<video_id>/rentals",methods=["GET"])
+def current_rental_customers(video_id):
+   
+    video_query=Video.query.get(video_id) #video record of video id
+   
+    customer_list=video_query.customers #list of customers that rented this video;accessing customer objects from query
+    rentals_response=[]
+    for customer in customer_list:
+        
+        rental_model=Rental.query.get((customer.id,video_id)) #
+
+        rentals_response.append(rental_model.get_rentals_by_video())
+    
+    return jsonify(rentals_response)
+
+
+@customer_bp.route("/<customer_id>/rentals",methods=["GET"])
+def current_rental_customers_by_customer(customer_id):
+    customer_query=Customer.query.get(customer_id)
+    video_list=customer_query.videos
+    rentals_response=[]
+    for video in video_list:
+        rental_model=Rental.query.get((customer_id,video.id))
+        rentals_response.append(rental_model.get_rentals_by_customers())
+
+    return jsonify(rentals_response)
+
```

Comment:
Great job in finishing all of the routes. As a future refactor, you can separate customer, video, and rental routes into separate files inside a 'routes' folder.